commons/pom.xml

```diff
@@ -161,9 +161,9 @@
 		</dependency>
 
 		<dependency>
-			<groupId>com.esotericsoftware</groupId>
+			<groupId>com.googlecode</groupId>
```

Comment:
This looks like a good change, but was it meant to be part of this pull request?